flair/datasets/sequence_labeling.py

```diff
@@ -562,6 +562,68 @@ def __init__(
             document_separator_token="-DOCSTART-",
             **corpusargs,
         )
+        
+        
+class STACKOVERFLOW_NER(ColumnCorpus):
+    def __init__(
+            self,
+            base_path: Union[str, Path] = None,
+            tag_to_bioes: str = "NE",
+            in_memory: bool = True,
+            document_as_sequence: bool = False,
+            **corpusargs,
+    ):
+        """
+        Initialize the STACKOVERFLOW_NER corpus. The first time you call this constructor it will automatically
+        download the dataset.
+        :param base_path: Default is None, meaning that corpus gets auto-downloaded and loaded. You can override this
+        to point to a different folder but typically this should not be necessary.
+        :param tag_to_bioes: NER by default, need not be changed, but you could also select 'pos' to predict
+        POS tags instead
+        :param in_memory: If True, keeps dataset in memory giving speedups in training.
+        :param document_as_sequence: If True, all sentences of a document are read into a single Sentence object
+        """
+        if type(base_path) == str:
+            base_path: Path = Path(base_path)
+
+
+        """The Datasets are represented in the Conll format.
+           In this format each line of the Dataset is in the following format:
+           <word>+"\t"+<NE>"\t"+<word>+"\t"<markdown>
+           The end of sentence is marked with an empty line.
+           In each line NE represented the human annotated named entity 
+           and <markdown> represented the code tags provided by the users who wrote the posts.
+           """
+        # column format
+        columns = {0: "word", 1: "NE", 2: "word", 3: "markdown"}
+
+        # this dataset name
+        dataset_name = self.__class__.__name__.lower()
+
+        # default dataset folder is the cache root
+        if not base_path:
+            base_path = Path(flair.cache_root) / "datasets"
+        data_folder = base_path / dataset_name
+
+        # download data if necessary
+        STACKOVERFLOW_NER_path = "https://raw.githubusercontent.com/jeniyat/StackOverflowNER/master/resources/annotated_ner_data/StackOverflow/"
+        cached_path(f"{STACKOVERFLOW_NER_path}train.txt", Path("datasets") / dataset_name)
+        cached_path(f"{STACKOVERFLOW_NER_path}test.txt", Path("datasets") / dataset_name)
+        cached_path(f"{STACKOVERFLOW_NER_path}dev.txt", Path("datasets") / dataset_name)
+        cached_path(f"{STACKOVERFLOW_NER_path}train_merged_labels.txt", Path("datasets") / dataset_name)
```

Comment:
Why is this file downloaded? It does not seem to be used. Also what is the difference between this file and train.txt?